### PR TITLE
Reuse html through SSI calls

### LINKED ISSUE #4: Dynamic loading of header + footer
We shouldn't need to replicate all of that html, it should just be loaded dynamically. Not sure the best approach for this, but I think we're already using jquery - jquery has an alright solution.

### PR DESCRIPTION
**Pull request by Mingling94**
Fixes #4 

Lots of copy/paste was needed to prevent future copy/paste on this code haha

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -86,66 +65,15 @@
...
-		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-		ga('create', 'UA-58715089-1', 'auto');
-		ga('send', 'pageview');
-	</script>
+
+		<!--#include file="/ssi/backgroundfooter.html" -->
+
+		<!--#include file="/ssi/googleanalytics.html" -->
```
Comment (DavidVorick) [inline about.html]: I prefer to name this 'analytics.html'

Comment (DavidVorick) [inline about.html]: also, is it okay to smash these together?

```
<!--#include ... -->
<!--#include ... -->
```

 As opposed to having a break between them.

**Comment by DavidVorick** [comment]
I chose to merge Adam's PR first, you'll have to move all this stuff to inside of public_html

**Comment by Mingling94** [comment]
Done!

**Pull request merged**